github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -481,88 +481,6 @@ periodics:
         path: /var/log/audit
         type: Directory
       name: audit
-- annotations:
-    k8s.v1.cni.cncf.io/networks: multus-cni-ns/sriov-passthrough-cni,multus-cni-ns/sriov-passthrough-cni
-    testgrid-dashboards: kubevirt-periodics
-    testgrid-days-of-results: "60"
-  cluster: prow-workloads
-  cron: 0 22, 10 * * *
-  decorate: true
-  decoration_config:
-    grace_period: 30m0s
-    timeout: 4h0m0s
-  extra_refs:
-  - base_ref: main
-    org: kubevirt
-    repo: kubevirt
-  labels:
-    preset-dind-enabled: "true"
-    preset-docker-mirror-proxy: "true"
-    preset-shared-images: "true"
-  name: periodic-kubevirt-e2e-k8s-1.22-sriov
```

Comment:
you might want to remove kind-1.22-sriov then as well please

Reply:
Yes I was going to check with you and @ormergi before removing the provider from kubevirtci